sysdeps/nacl/alarm.c

```diff
@@ -0,0 +1,21 @@
+#include <unistd.h>
+#include <sys/time.h>
+
+unsigned int alarm(unsigned int seconds) {
+  struct itimerval timer;
+  struct itimerval old_timer;
+
+  timer.it_value.tv_sec = seconds;
+  timer.it_value.tv_usec = 0;
+  timer.it_interval.tv_sec = 0;
+  timer.it_interval.tv_usec = 0;
+
+  setitimer(ITIMER_REAL, &timer, &old_timer);
```

Comment:
did you confirm this routes to your settimer function?

Reply:
shouldn't it be the lind one?

Reply:
Adding some print statements shows that the setitimer I implemented is called when calling alarm.